frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -12,11 +12,23 @@ import { addIDPItems } from './oauth';
 import { TextFilter } from '../factory';
 import { fuzzyCaseInsensitive } from '../factory/table-filters';
 import { withExtensions, isGlobalConfig, GlobalConfig } from '@console/plugin-sdk';
+import i18next from 'i18next';
 
 const stateToProps = (state: RootState) => ({
   configResources: state.k8s.getIn(['RESOURCES', 'configResources']),
 });
 
+export const breadcrumbsForGlobalConfig = (detailsPageKind: string, detailsPageUrl: string) => [
```

Comment:
This should say "path" instead of "url"

```suggestion
export const breadcrumbsForGlobalConfig = (detailsPageKind: string, detailsPagePath: string) => [
```

Reply:
fixed